--- miniunz1.c	2022-05-25 10:44:39.594027054 +0200
+++ miniunz.c	2022-05-25 10:44:45.194041662 +0200
@@ -247,8 +247,8 @@
     printf("  ------  ------     ---- -----   ----    ----   ------     ----\n");
     for (i=0;i<gi.number_entry;i++)
     {
-        char filename_inzip[256];
-        unz_file_info64 file_info;
+        char filename_inzip[256] = {0};
+        unz_file_info64 file_info = {0};
         uLong ratio=0;
         const char *string_method;
         char charCrypt=' ';
